lib/main_containers.pm

```diff
@@ -100,22 +100,33 @@ sub load_host_tests_load_containerd_nerdctl {
 
 sub load_container_tests {
     my $runtime = get_required_var('CONTAINER_RUNTIME');
+    $runtime =~ tr/ //ds;
+    my @runtimes = split(',', uc $runtime);
+
```

Comment:
```
use Data::Dumper;
my $r1 = "a,b, c,d,      e";
foreach (split(',\s*', $r1)) {
        print "token .... " . Dumper($_);
}
```

Works nice too

```
token .... $VAR1 = 'a';
token .... $VAR1 = 'b';
token .... $VAR1 = 'c';
token .... $VAR1 = 'd';
token .... $VAR1 = 'e';
```

I admit you would filter out all whitespaces :)

Reply:
Now this is implemented, thanks both